Core/Src/main.c

```diff
@@ -42,13 +42,19 @@
 /* USER CODE END PM */
 
 /* Private variables ---------------------------------------------------------*/
+SPI_HandleTypeDef hspi;
+TIM_HandleTypeDef htim1;
 
 /* USER CODE BEGIN PV */
-
+uint16_t Size = 10;
+uint32_t Timeout = HAL_MAX_DELAY;
+uint16_t Period = 60000;
```

Comment:
Could you also push all other files that has been auto-generated by the stm ide? That is also part of this bootcamp, I also need to see those files to check the correctness of your configuration.

Reply:
I would recommend writing the variables that you think will not be changed in the following use as a `const` and name it with all capitalized letters